paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -157,8 +158,26 @@ private fun SemanticsNode.accessibilityText(): String? {
   val role = config.getOrNull(SemanticsProperties.Role)?.toString()
   val disabled = if (config.getOrNull(SemanticsProperties.Disabled) != null) "<disabled>" else null
   val onClickLabel =
-    if (disabled != null) null else config.getOrNull(SemanticsActions.OnClick)?.label?.let { "<on-click>: $it" }
+    if (disabled != null) {
+      null
+    } else {
+      config.getOrNull(
+        SemanticsActions.OnClick
+      )?.label?.let { "<on-click>: $it" }
```

Comment:
is this not able to fit on a single line?

Reply:
this change was from running spotless